packages/ui/src/i18n/dictionaries/authenticator/sv.ts

```diff
@@ -20,7 +20,7 @@ export const svDict: AuthenticatorDictionary = {
   'Dismiss alert': 'Avvisa varning',
   Email: 'E-post',
   'Enter your code': 'Skriv din kod',
-  'Enter your Email': 'Fyll i din e-post',
+  'Enter your email': 'Ange din e-post',
```

Comment:
```suggestion
  'Enter your Email': 'Ange din e-post',
```
Within our current testing, "Email" is case sensitive for this field

Reply:
Good find, thanks!